crates/bevy_pbr/src/lib.rs

```diff
@@ -186,12 +186,20 @@ impl Default for PbrPlugin {
 }
 
 /// A resource that stores the spatio-temporal blue noise texture.
-#[derive(Resource)]
+#[derive(Resource, Clone)]
 pub struct Bluenoise {
     /// Texture handle for spatio-temporal blue noise
     pub texture: Handle<Image>,
 }
 
+impl FromWorld for Bluenoise {
```

Comment:
Could we make this a system rather than a FromWorld impl? Something like `init_blue_noise`

Reply:
it seemed to be pretty sensitive to ordering, i dont want to figure out porting this to RenderStartup in this PR

Reply:
Yeah, I'm okay with porting it later

Reply:
Removed this declaration for `FromWorld` and put the code block back into PBR plugin with a conditional feature flag.